packages/core/src/messages/responses/cache-set.ts

```diff
@@ -1,16 +1,16 @@
 import {SdkError} from '../../errors';
 import {BaseResponseError, BaseResponseSuccess} from './response-base';
-import {MomentoResponse} from './enums';
+import {CacheSetResponse} from './enums';
 
 interface IResponse {
-  type: MomentoResponse;
+  type: CacheSetResponse;
```

Comment:
We had a slack discussion about this. Having a generic `MomentoResponse` for apis that return `Success` or `Error`, but having api specific enums for apis that return `Hit` `Miss` or `Error` was not consistent. We are giving each api their own unique enum, as that gives us the most flexibility moving forward. If we need to add a third return type to the `Set` api in the future, we just need to change this one enum, and not worry about affecting other apis

Reply:
what you described above is the correct approach, ty